src/renderers/dom.js

```diff
@@ -176,6 +198,9 @@ export function dom(opts? : DomOptions = getDefaultDomOptions()) : DomRenderer {
         }
         
         if (node.type === NODE_TYPE.ELEMENT) {
+            if (node.name.toLowerCase() === 'svg' || typeof node.props.xmlns !== 'undefined') {
```

Comment:
Is the intention here to add `xmlns` to *every* element which does not already have it set?

Reply:
No. I need to refactor this line. 

The intention is that every `<svg>`  element have a namespace, and that all of its children either have a namespace, or fallback to the namespace of the parent `<svg>`. I was trying to create a more elegant solution than a regex to match every possible svg child element (`match('path|rect|circle|use|polygon|text|g|style...... ect')` ), but I have not landed on a good way to test for it yet.